tests/platform/linux/linux-builtIn-testSpec.js

```diff
@@ -30,7 +30,9 @@ gpii.tests.linux.builtIn = [
                 "some.app.id": [{
                     "settings": {
                         "mag-factor": 1.5,
-                        "screen-position": "full-screen"
+                        "screen-position": "full-screen",
+                        "mouse-tracking": "proportional",
+                        "caret-tracking": "proportional"
```

Comment:
To make sure that the transform is properly tested, could you add the case for focus tracking not being part of the NP set -- ie:

"focus-tracking": "none" 

(for both test cases)